examples/openshift.json

```diff
@@ -0,0 +1,67 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "OpenShift",
+      "openShiftConfig": {
+        "clusterUsername": "demo",
+        "clusterPassword": "demo"
+      }
+    },
+    "azProfile": {
+      "tenantId": "YOUR TENANT ID",

```

Comment:
nit: the current idiom is to leave priviileged values (or properties for which it doesn't make sense to have default example vals) blank in example api models, e.g., `"tenantId": "",`

Reply:
https://github.com/kargakis/acs-engine/issues/99 as a follow-up

Reply:
ACK